src/form.php

```diff
@@ -119,7 +120,7 @@
                         </div>
                         <div class="form-group col-md">
                             <label for="email"><?=_("Email")?> *</label>
-                            <input type="email" class="form-control" id="email" required>
+                            <input type="email" class="form-control" id="email" name="email" required>
```

Comment:
Ahí tiene más sentido conservar solo la etiqueta name

```suggestion
                            <input type="email" class="form-control" name="email" required>
```